packages/backend/.env

```diff
@@ -24,4 +24,3 @@ AP_WARNING_TEXT_HEADER=""
 AP_FLOW_WORKER_CONCURRENCY="5"
 
 AP_DEV_PIECES="discord,delay,hackernews,github,slack,store,wordpress"
```

Comment:
why remove this ?

Reply:
It doesn't work in the development mode.